$NetBSD$

Force our libltdl.
Fix flawed logic: "/proc/cpuinfo" isn't linux-specific.

--- configure.orig	2010-03-31 05:36:48.000000000 +0400
+++ configure	2010-03-31 20:48:43.000000000 +0400
@@ -74110,7 +74110,7 @@
 fi
 
    case "${host}" in
-   i?86-*|x86_64*|ia64-*|powerpc-*|powerpc64-*|sparc*-*)
+   i?86-*linux*|x86_64*linux*|ia64-*linux*|powerpc-*linux*|powerpc64-*linux*|sparc*-*linux*)
         if test "$timer_linux_happy" = "yes"; then :
   if test -r "/proc/cpuinfo"; then :
   timer_linux_happy="yes"
@@ -141912,9 +141912,10 @@
   "") enable_ltdl_convenience=yes
       ac_configure_args="$ac_configure_args --enable-ltdl-convenience" ;;
 esac
-LIBLTDL='${top_build_prefix}'"${lt_ltdl_dir+$lt_ltdl_dir/}libltdlc.la"
+#LIBLTDL='${top_build_prefix}'"${lt_ltdl_dir+$lt_ltdl_dir/}libltdlc.la"
+#LIBLTDL=libltdl.la
 LTDLDEPS=$LIBLTDL
-LTDLINCL='-I${top_srcdir}'"${lt_ltdl_dir+/$lt_ltdl_dir}"
+#LTDLINCL='-I${top_srcdir}'"${lt_ltdl_dir+/$lt_ltdl_dir}"
 
 
 
@@ -157040,7 +157041,7 @@
         CPPFLAGS="-I."
         cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
-#include <opal/libltdl/ltdl.h>
+#include <ltdl.h>
 
 _ACEOF
 if (eval "$ac_cpp conftest.$ac_ext") 2>&5 |
